### ISSUE TITLE
Application connection requests tab behaviour

### ISSUE THREAD

**Issue by mrvanes** [SURFscz/SBS#1677]
#1663 was checked and accepted to work as designed.

However, isn't it strange that an organisation admin _loses_ the ability to monitor the acceptance of the application when service connection has to be approved? And what's the difference between an immediately accepted application connection request and one that needed application admin approval? In the end, they result in a connected application and I would expect both paths to lead to the same interface result?

**Comment by logan-life** [comment]
agreed, the current behavior is a bit strange. will think about this.

it would also be good to inform the user about the status of their request, e.g. "awaiting organisation approval" and "awaiting application owner approval"

will refine this. thanks martin!

**Comment by logan-life** [comment]
todo: inform the end user about the status of their request (incl. "awaiting org approval" and "awaiting application approval") and not yet the informing org admin part. logan to refine this a bit more.

**Comment by EricvanSchaik** [comment]
@logan-life is this still on your radar?

**Comment by logan-life** [comment]
Yup!
